src/files/FilesPage.js

```diff
@@ -142,13 +89,12 @@ class FilesPage extends React.Component {
       root = files.path
     }
 
-    const updater = this.makeUpdater('addProgress')
-    await doFilesWrite(root, raw, updater)
+    await doFilesWrite(root, raw)
   }
 
-  addByPath = (path) => {
+  addByPath = async (path) => {
     const { doFilesAddPath, files } = this.props
-    doFilesAddPath(files.path, path)
+    await doFilesAddPath(files.path, path)
```

Comment:
`await` but not using the return value, or catching errors. We shouldn't rely on the return values from redux actions... actions update the store and we render that. I should be fine to remove the `await`

Reply:
Yes!